Add target_index_settings to rollup documentation (#9302)
opensearch-project/documentation-website: _im-plugin/index-rollups/rollup-api.md

:--- |:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|:--- |:--- |
`source_index` | The name of the detector.                                                                                                                                                                                                                                                                                                                                               | String | Yes
`target_index` | Specify the target index that the rolled up data is ingested into. You can either create a new target index or use an existing index. The target index cannot be a combination of raw and rolled up data. This field supports dynamically generated index names like {% raw %}`rollup_{{ctx.source_index}}`{% endraw %}, where `source_index` cannot contain wildcards. | String | Yes

> `target_index_settings` | You can specify any [index settings]({{site.url}}{{site.baseurl}}/im-plugin/index-settings/), that will be applied if the target index is created during the rollup.                                                                                                                                                                                                    | Object | No

Reviewer: ```suggestion
`target_index_settings` | Specify any [index settings]({{site.url}}{{site.baseurl}}/im-plugin/index-settings/) to be applied to the target index created during the rollup.                                                                                                                                                                                                    | Object | No
```

Author: Thank you for review @kolchfa-aws !
